pkg/models/resources/v1alpha2/storageclass/storageclasses.go

```diff
@@ -85,9 +91,11 @@ func (s *storageClassesSearcher) Search(namespace string, conditions *params.Con
 	r := make([]interface{}, 0)
 	for _, i := range result {
 		count := s.countPersistentVolumeClaims(i.Name)
+		isSnapshotAllow := s.isSnapshotAllow(i.Provisioner)
 		if i.Annotations == nil {
 			i.Annotations = make(map[string]string)
 			i.Annotations["kubesphere.io/pvc-count"] = string(count)
+			i.Annotations["kubesphere.io/allowSnapshot"] = strconv.FormatBool(isSnapshotAllow)
```

Comment:
conventionally, we use `kubesphere.io/allow-snapshot` instead of camelCase as annotation name

Reply:
ok